src/routes/(admin sponsor)/sponsor-admin/my-network/+layout.js

```diff
@@ -1,20 +0,0 @@
-import auth0 from '$lib/security/client';
-import lodash from 'lodash';
-
-import partnerNetworkApi from '$dataSources/api.that.tech/partner/leads/queries';
-
-const { sortBy } = lodash;
-
-export const load = auth0.withPageAuthRequired({
```

Comment:
I don't see where we're checking for a session on this route anylonger.

Reply:
`authorization` function in `hooks.server.js`